templates/github/.ci/ansible/Containerfile.j2.copy

```diff
@@ -27,7 +27,11 @@ RUN pip3 install \
 {{ " " }}"{{ item.source }}"
 {%- endfor %}
 
-RUN mkdir -p /etc/nginx/pulp/
+USER pulp:pulp
+RUN PULP_STATIC_ROOT=/var/lib/operator/static/ PULP_CONTENT_ORIGIN=localhost \
+       /usr/local/bin/pulpcore-manager collectstatic --clear --noinput --link
+USER root:root
```

Comment:
Do we want to change back to `root:root`?

Reply:
What are changing back to root:root is the context under which all subsequent commands like "RUN" in the Containerfile run as.

We want the rest of the Containerfile, including the symlinking, to do that.